testsuite/tests/tool-toplevel/pr7060.ml.reference

```diff
@@ -1,16 +0,0 @@
-
-# type t = A | B
-# type u = C of t
-# Characters 18-54:
-  let print_t out = function A -> Format.fprintf out "A";;
-                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Warning 8: this pattern-matching is not exhaustive.
-Here is an example of a case that is not matched:
-B
-val print_t : Format.formatter -> t -> unit = <fun>
-# # - : t =
-<printer print_t raised an exception: File "//toplevel//", line 1, characters 18-23: Pattern matching failed>
```

Comment:
This seems to be missing from the new `.reference` file, is it really testing the same thing? [MPR#7060](https://caml.inria.fr/mantis/view.php?id=7060) is about exceptions raised by the printers, and there is no trace of exceptions in pr7060.compilers.reference.